features/help.js

```diff
@@ -41,6 +41,13 @@ The more emojis you add, the more recognition they get!
 
 > @alice just pushed the cleanest code I've ever seen! ${recognizeEmoji} ${recognizeEmoji} ${recognizeEmoji}
 
+Use multipliers to give more recognition!
```

Comment:
Great catch, didn't even realize we never included a multiplier example in the help message 😅